cmd/secrets.go

```diff
@@ -26,11 +25,21 @@ var (
 		Args:         cobra.MaximumNArgs(1),
 		SilenceUsage: true,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			if err := validateSecretCommandInput(args); err != nil {
+			if err := validateSecretCommandInput(); err != nil {
 				cmd.Usage()
 				return err
 			}
-			return executeSecretCleanupCommand(args)
+
+			coreClient, err := kubernetes.NewCoreV1Client()
+			if err != nil {
+				return fmt.Errorf("cannot initiate kubernetes core client")
+			}
+
+			secretService := service.SecretService{
+				Secret: coreClient.Secrets(config.Namespace),
+				Helper: kubernetes.New(),
+			}
```

Comment:
Usually in Go we provide new static Constructor in the "secret" package, e.g. `NewSecretsService(client, helper)`